projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  <button mat-button (click)="OpenInApp()">
```

Comment:
based on Angular guidelines, method names should start with lowercase

Reply:
needs to be tested

Reply:
noted and resolved